modules/ROOT/pages/mmp-concept.adoc

```diff
@@ -775,15 +805,24 @@ Below is an example of a CloudHub Deployment using encrypted credentials:
 
 NOTE: Make sure that username and password are not set in the deployment configuration, as they have precedence over the defined server id.
 
-== Mule Maven Plugin Goals description
+== Mule Maven Plugin Goals Description
 
-=== Package Goal
+The Mule Maven Plugin has three goals that are explained below: Package, Deploy and Undeploy.
+
+Each goal accepts different parameters that change the plugin behavior. When you provide a parameter from the command line, you have to prepend `-D` to the parameter name.
+
+For example, to execute the `package` goal and set the `attachMuleSources` parameter, you run the following command:
+[source,console,linenums]
+----
+mvn package -DattachMuleSources
+----
 
-Generates the application JAR file.
-Bind's by default to the http://maven.apache.org/ref/3.6.1/maven-core/lifecycles.html[lifecycle phase]: package
+=== Package Goal
 
-Optional parameters:
+This goal generates the application JAR file.
+Bind's by default to the http://maven.apache.org/ref/3.6.1/maven-core/lifecycles.html[lifecycle phase]: `package`
```

Comment:
Is this supposed to be part of or a followup to the previous sentence, stating that the goal "binds" (not "bind's") to the lifecycle phase? What is the ": `package`" syntax? If it is at the end of the sentence, if should be followed by a period, but I am not sure that this is a complete sentence, as-is....